algorithms/CPlusPlus/Stacks/prefix_to_postfix.cpp

```diff
@@ -0,0 +1,64 @@
+// Program to convert a prefix expression to postfix expression
+// Prefix expression : An expression in which the operator appears before the operands. 
+// Postfix expression : An expression in which the operator appears after the operands. 
+
+
+
+// Alogorithm
+// To convert prefix to postfix expression we follow steps given below:
+// Step 1. If the symbol is an operand, push it to the stack
+// Step 2. If symbol is an operator then pop top two elements from stack
+// Step 3. Then push an expression by concatenating (+op1+op2+symbol)
+// Step 4. At last return the top element of stack as postfix expression
+
+
+
+// CODE:
+#include<iostream>
+#include<bits/stdc++.h>
+using namespace std;
+
+// Function to convert prefix to postfix expression
+string prefix_to_postfix(string prefix)
+{
+    stack<string> s;
+    // length of prefix expression
+    int l=prefix.length();
+  
+    // Iterating through the Prefix expression from right to left
+    for(int i=l-1;i>=0;i--){
+        // if the symbol is an operand, push it to the stack
+        if(prefix[i]>='a' && prefix[i]<='z'||prefix[i]>='A' && prefix[i]<='Z')
+        s.push(string(1,prefix[i]));
```

Comment:
```suggestion
        if(prefix[i]>='a' && prefix[i]<='z'||prefix[i]>='A' && prefix[i]<='Z'){
            s.push(string(1,prefix[i]));
        }
```
not a good practice to write like that, if you miss brackets, sometimes it will come around and give you a big headache during debugging.